### DISCUSSION TITLE
German BSI Project CAOS security issues

### DISCUSSION THREAD

**Discussion by clemone210** [dani-garcia/vaultwarden#5120] [General]
Hi there,
I could not find any specific discussions or issues within this repo, so Id like to ask if the security concerns raised by the German BSI project CAOS have already been addressed?
Could you provide some information on this?

https://www.bsi.bund.de/DE/Service-Navi/Publikationen/Studien/Projekt_P486/projekt_P486_node.html
[Vaultwarden-Passwortmanager.pdf](https://github.com/user-attachments/files/17473085/Vaultwarden-Passwortmanager.pdf)

**Comment by BlackDex** [comment]
All issues are already fixed, you can find the reported CVE's here:
https://cve.mitre.org/cgi-bin/cvekey.cgi?keyword=Vaultwarden

And you can find them in our repo here:
https://github.com/dani-garcia/vaultwarden/pulls?q=is%3Apr+cve+mitre+is%3Aclosed+

Also some none-cve reported issues were addressed.

**Reply by clemone210** [reply]
Thanks for the quick reply.
I didn't look for those search parameters, thanks for pointing them out!
